pkg/pool-stable-phantom/test/StablePhantomPool.test.ts

```diff
@@ -76,6 +76,8 @@ describe('StablePhantomPool', () => {
     const tokenRateCacheDurations: number[] = [];
     const exemptFromYieldProtocolFeeFlags: boolean[] = [];
 
+    const ZEROS = Array(numberOfTokens + 1).fill(bn(0));
+
     async function deployPool(
```

Comment:
Something I noticed here is that we're deploying a rate for each token so we never test the case where a token doesn't have a rate provider to call.

Reply:
Yes; addressing as part of the test I'm adding for the 0 rate provider with exempt flag set